src/senaite/core/browser/controlpanel/manufacturers/view.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+import collections
+
+from bika.lims import api
+from bika.lims import bikaMessageFactory as _
+from bika.lims.utils import get_link_for
+from senaite.app.listing import ListingView
+from senaite.core.catalog import SETUP_CATALOG
+
+
+class ManufacturersView(ListingView):
+    """Displays all available manufacturers
+    """
+
+    def __init__(self, context, request):
+        super(ManufacturersView, self).__init__(context, request)
+
+        self.catalog = SETUP_CATALOG
+
+        self.contentFilter = {
+            "portal_type": "Manufacturer",
+            "sort_on": "sortable_title",
+        }
+
+        self.context_actions = {
+            _("Add"): {
+                "url": "++add++Manufacturer",
+                "icon": "++resource++bika.lims.images/add.png",
+            }}
+
+        t = self.context.translate
+        self.title = t(_("Manufacturers"))
+        self.description = ""
+        self.icon = "{}/{}".format(
+            self.portal_url,
+            "/++resource++bika.lims.images/manufacturer_big.png"
+        )
+
+        self.show_select_row = False
+        self.show_select_column = True
+        self.show_all = False
+
+        self.columns = collections.OrderedDict((
+            ("Title", {
+                "title": _("Title"),
+                "index": "sortable_title"}),
```

Comment:
Please use the message factories with our `i18n` ID here (and for `Description` as well. E.g.

```python
"title": _(
    u"listing_manufacturers_column_title",
    default="Title"
)
```